### ISSUE TITLE
Logo creation Aragon DAC

### ISSUE THREAD

**Issue by krrisis** [aragon/aragondac-community#1]
- [ ] briefing by @quazia
- [ ] feedback by griff, maria & others on v1
- [ ] new version by  23/07 AM
- [ ] OK by Quazia Griff Chris Luke by 23/07 EOD 
- [ ] Send final version in jpg, png, ai & pdf by 24/07 EOD
- [ ] Receive estimation ?
- [ ] Payment after reception of funds

**Comment by krrisis** [comment]
[propose_02_v01.pdf](https://github.com/aragon/aragondac-community/files/2210400/propose_02_v01.pdf)
the two initial proposals

**Comment by luisivan** [comment]
OMG that is so beautiful!!

**Comment by Smokyish** [comment]
That is lovely, really like 02 v01 a lot.

@krrisis How is this progressing? I see that the dates on the checklist have gone already?

**Comment by krrisis** [comment]
@smokyish yup, Quazia was on this but went MIA - contacted the designer directly, new proposal coming up this evening if all goes well

**Comment by krrisis** [comment]
Updated logo here! https://matrix.to/#/!mdWycbiXAbZetTcORz:matrix.org/$153323939354738zhovy:matrix.org

**Comment by krrisis** [comment]
Final logo [here](https://drive.google.com/open?id=13cP3xw9PT8qnz2m5JMD9xsc62T1DHzt_) ! 
Milestone for Adrian's great work made as well - will be paid as soon as budget is sent.

**Comment by krrisis** [comment]
Logos also added to yr site by @Smokyish <3 https://wiki.aragon.org/design/logo/#aragon-dac

**Comment by john-light** [comment]
Is this issue ready to be closed?

**Comment by krrisis** [comment]
Yup! Link to the Milestone [here](https://beta.giveth.io/campaigns/5b5b38b19737b23ee71c73a6/milestones/5b69e4340fdcbf1a7c8efef9)
